dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -511,6 +565,32 @@ struct LoadArgs {
     collection_id: Option<CollectionUuid>,
 }
 
+#[derive(Debug, Args)]
+struct LoadExampleArgs {
+    /// Seed for the example system.
+    ///
+    /// The seed is used to generate a deterministic example system. For tests,
+    /// the seed is typically the name of the test.
```

Comment:
```suggestion
    /// Seed for the RNG that's used to generate the example system.
    ///
    /// Setting this makes it possible for callers to get deterministic results.  In automated tests, the seed is typically the name of the test.
```

(clarifying because sometimes we use "seed" to mean "template" -- this is not that)

Reply:
Done.